app/static/js/searchOutsideParticipant.js

```diff
@@ -0,0 +1,31 @@
+export default function searchOutsideParticipant(inputId, callback, parentElementId=null){
+  var query = $(`#${inputId}`).val()
+  console.log("searchOutsideParticipant is invoked here");
```

Comment:
We have to remove the debug statements.

Reply:
Also, this function is very similar to the searchUser one. Can't you use the same javascript file (I believe it's called searchUser)? 
Why the need to define it again?